source/loader/ur_lib.cpp

```diff
@@ -174,4 +182,14 @@ ur_result_t urLoaderConfigEnableLayer(ur_loader_config_handle_t hLoaderConfig,
     hLoaderConfig->enabledLayers.insert(pLayerName);
     return UR_RESULT_SUCCESS;
 }
+
+ur_result_t urLoaderTearDown(void *pParams) {
+    if (!pParams) {
```

Comment:
do we know what the intent of this parameter was? since we don't seem to be using it and now no adapters can use it maybe we should look into getting rid of it or making it optional

Reply:
I'm not sure what the parameter was meant to be for - @pbalcer do you know?
I suspect it's not needed, the documentation for `ur_tear_down_params_t` mentions parameters for callbacks, but I don't think any teardown callbacks actually exist

Reply:
Maybe it's in case we need to add an argument for teardown in future releases?
Generic callbacks were removed sometime back. I vote we remove it.

Reply:
I've removed it, we can always re-add it in future if an actual use becomes apparent